packages/plugin-docusaurus/src/client/theme/SearchBar/index.tsx

```diff
@@ -109,7 +132,16 @@ export default function SearchBar(): JSX.Element {
         render(
           <>
             <div className="aa-PanelLayout aa-Panel--scrollable">{sections}</div>
-            <Footer />
+            <Footer colorMode={colorMode} />
+          </>,
+          root
+        )
+      },
+      renderNoResults({ render, state }, root) {
+        render(
+          <>
+            {state.query && <div className="aa-NoResults">No results found.</div>}
```

Comment:
Not sure `state.query` is the right value that we should leverage. Is it correct?

Reply:
AFAIK I got from the widget documentation yes.